src/collection/views/Collection.tsx

```diff
@@ -302,6 +316,154 @@ const Collection: FunctionComponent<CollectionProps> = ({ match, history }) => {
 				<Container mode="vertical">
 					<Container mode="horizontal">{renderContentBlocks(contentBlockInfos)}</Container>
 				</Container>
+				<Container mode="vertical">
+					<Container mode="horizontal">
+						<h3 className="c-h3">Info over deze collectie</h3>
+						<Grid>
+							<Column size="3-3">
+								<Spacer margin="top">
+									<p className="u-text-bold">Onderwijsniveau</p>
+									<p className="c-body-1">
+										{collection.lom_context && collection.lom_context.length ? (
+											(collection.lom_context || []).map((lomContext: string) =>
+												generateSearchLinks(String(collection.id), 'educationLevel', lomContext)
+											)
+										) : (
+											<div>-</div>
```

Comment:
`Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>` hier mss best een `<span>` van maken. Als die op `display: block;` moet staan kan je deze className toevoegen: `u-d-block`